scripts/gen_help_html.lua

```diff
@@ -1440,7 +1440,6 @@ function M.test_gen(help_dir)
     -- Because gen() is slow (~30s), this test is limited to a few files.
     { 'help.txt', 'index.txt', 'nvim.txt' }
   )
-  eq(3, #rv.helpfiles)
```

Comment:
was this intentionally removed? the purpose is to positively confirm that something is actually being tested. otherwise it's possible that `eq(0, err_count)` (below) could be a false negative.

Reply:
Yes, intentional. It seems useless.

> the purpose is to positively confirm that something is actually being tested.

Don't know what you mean. We're asserting that the length of the table is the length of the table.

Reply:
We can rewrite this as `eq(vim.tbl_count(rv.helpfiles), #rv.helpfiles)` if you like that more?

Reply:
That's a _different_ table than the one on line 1441, though (output, not input).

Reply:
In that case we should be comparing the actual length of the input table as to not need to manually update this value every time we remove or add an element rather than hardcode it. There is no way to know the intention of what `3` is supposed to represent without prior knowledge.

Reply:
That makes sense, yes, and would be a useful change.

Reply:
https://github.com/neovim/neovim/pull/28976